.circleci/config.yml

```
@@ -224,6 +227,11 @@ jobs:
             key: front-packages-shared-{{ checksum "~/shared.hash" }}
             paths:
                 - << parameters.path_to_front_packages >>/shared/
+      - save_cache:
+          name: Save micro-frontend Catalog Volume Monitoring cache
+          key: micro-frontend-catalog-volume-monitoring-{{ checksum "~/catalog-volume-monitoring.hash" }}
+          paths:
+            - src/Akeneo/Platform/Bundle/CatalogVolumeMonitoringBundle/front
```

As the CE can launch a CE build we need to use path_to_front_packages in order to retrieve the path depending on witch version you are

```suggestion
            - << parameters.path_to_front_packages >>/../src/Akeneo/Platform/Bundle/CatalogVolumeMonitoringBundle/front
```